web/backend/src/cli.ts

```diff
@@ -65,4 +66,15 @@ program
     console.log('Permissions removed successfully!');
   });
 
+program
+  .command('keygen')
+  .description('Create a new keypair for the .env')
+  .action(() => {
+    const ed25519 = curve.newCurve('edwards25519');
+    const priv = ed25519.scalar().pick();
+    const pub = ed25519.point().mul(priv);
+    console.log(`PRIVATE_KEY=${priv}`);
```

Comment:
is it normal to simply print them instead of saving them where they are needed?

Reply:
The goal here is to create the values that are needed for the `.env` file, so not to store them anywhere.

Reply:
maybe add a line saying "please make sure to store this somewhere" as people might indeed expect the values to also be written to a file

Reply:
Added an output, but now it won't be usable in scripts anymore...